Guesser/DoctrineFieldGuesser.php

```diff
@@ -62,7 +62,8 @@ public function getAllFields($class)
     public function getManyToMany($model, $fieldPath)
     {
         $resolved = $this->resolveRelatedField($model, $fieldPath);
-        return !$this->getMetadatas($resolved['class'])->isAssociationWithSingleJoinColumn($resolved['field']);
+        $metadata = $this->getMetadatas($resolved['class']);
+        return $metadata->hasAssociation($resolved['field']) && !$metadata->isAssociationWithSingleJoinColumn($resolved['field']);
```

Comment:
We have to check if if association even exists, before we check if that is many-to-many association. 

Otherwise, this method would return "true" if association does not exists, because, if there is no association, it is not association with single join column -> which will get result "it is many-to-many" -> which is wrong.